### PR TITLE
List

### PR DESCRIPTION
**Pull request by Jia-A**
Screenshot (444)
Screenshot (445)
Screenshot (446)
Screenshot (447)
Added 4 types of lists for different uses.

### CODE REVIEW

**Review by hinanshisuthar — COMMENTED** [review]
Looks good to me.

File: components/list/list.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    list-style-type: circle;
+}
+.stacked, .spaced{
+    padding: 10px;
+    border-radius: var(--border-radius);
+    color: var(--primary-color);
+    background-color: var(--background-color);
+}
+.stacked{
+    margin: 15px;
```
Comment (shivanigangadharan) [inline components/list/list.css]: Could be a good practice in general to use rem for margin and padding.

**Review by shivanigangadharan — APPROVED** [review]
Nice work, can merge. 👍

File: components/list/list.css (hunk 1 above)
Comment (Jia-A) [reply components/list/list.css]: okay, might have forgotten to change here, will do it right away.

**Pull request merged**